src/Microsoft.ML.Recommender/Microsoft.ML.Recommender.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
This shouldn't be `netcoreapp2.0`.  It should be `netstandard2.0` instead.

Reply:
Fixed. Thanks.